include/arch/xtensa/arch.h

```diff
@@ -115,17 +115,12 @@ extern void _irq_priority_set(u32_t irq, u32_t prio, u32_t flags);
 #define _ARCH_IRQ_CONNECT(irq_p, priority_p, isr_p, isr_param_p, flags_p) \
 ({ \
 	enum { IRQ = irq_p }; \
-	static struct _isr_table_entry \
-		_CONCAT(_isr_irq, irq_p) \
-		__attribute__ ((used)) \
-		__attribute__ ((section(\
-			STRINGIFY(_CONCAT(.gnu.linkonce.d.isr_irq, irq_p)))\
-		)) = {isr_param_p, isr_p}; \
-	_irq_priority_set(irq_p, priority_p, flags_p); \
+	extern struct _isr_table_entry _sw_isr_table[]; \
+	_sw_isr_table[IRQ].isr = isr_p; \
```

Comment:
Figured out why objcopy was failing. The `OUTPUT_FORMAT` variable in the `gen_isr_tables` Makefile was different than what `objcopy` for the target expected.